openmc/data/effective_dose/photons_kerma.txt

```diff
@@ -0,0 +1,36 @@
+Photons: Effective dose per air kerma free-in-air, in units of Sv/Gy, for monoenergetic particles incident in various geometries.

```

Comment:
Yes, this is why I didn't include 'photon_kerma' as an option in the list. All these text files though are just directly dumped from the ICRP-116 supplemental data. If you think it would be less confusing, I can just remove this file?

Reply:
I could go either way on keeping or removing it, but I'd lean a little toward keeping it as it's a nice capability to have around. The part that's a little odd to me is that it's still possible to get coefficients for the photon kerma from `dose_coefficients` but it's restricted to only a few of the available geometries according to the data.

Reply:
As far as the coefficient units go, a wrapper function for `dose_coefficients` specifically for the photon kerma might make sense to separate intent?

Reply:
After offline discussion, a caveat has been added in the `dose_coeffcient` docstrings about the photon kerma units. Those values make sense to be returned from this function as they are still dose coefficients despite having different units of conversion.